api_tests/lib/actor.ts

```diff
@@ -192,6 +192,10 @@ export class Actor {
         if (network !== "regtest") {
             throw Error("Expected network regtest, found " + network);
         }
+
+        // ensure that both parties are watching for the action
```

Comment:
Could you make the comment a bit more verbose please?
I'd like to be able to understand whether or not I can remove the sleep from the comment in 1 year time.

Reply:
Sure. I created issue #1289 which includes reverting this PR anyway.

Reply:
Is it clearer now?